libs/sr/stats/src/char.ts

```diff
@@ -19,7 +19,16 @@ export function getCharStat(ck: CharacterGenderedKey) {
  */
 export function getCharInterpolateObject(
   ck: CharacterGenderedKey,
-  skType: 'basic' | 'skill' | 'ult' | 'talent' | 'technique' | 'eidolon',
+  skType:
+    | 'basic'
+    | 'skill'
+    | 'ult'
+    | 'talent'
+    | 'technique'
+    | 'eidolon'
+    | 'bonusAbility1'
```

Comment:
Seems possibly broken for bonus ability
![image](https://github.com/user-attachments/assets/e22b437a-e507-4d57-bfcd-7c037777aaca)

Reply:
Yeah, I noticed this. I added a `// FIXME`
https://github.com/frzyc/genshin-optimizer/blob/67fe5671582edf4f0c6e165038f49c128875398e/libs/sr/formula-ui/src/char/sheetUtil.tsx#L76

Reply:
I did not touch datamine stuff so I did not dig too deeply.